image/templates/common/port-forward.sh

```diff
@@ -16,20 +16,22 @@
 #     $ ./port-forward.sh 8443
 
 if [ -z "$1" ]; then
-	echo "usage: $0 8443"
+	echo "usage: $0 8443 [namespace]"
 	echo "The above would forward localhost:8443 to central:443."
 	exit 1
 fi
 
 KUBE_COMMAND=${KUBE_COMMAND:-{{.K8sConfig.Command}}}
+PORT="$1"
+NAMESPACE=${2:-stackrox}
 
 while true; do
-    central_pod="$(${KUBE_COMMAND} get pod -n 'stackrox' --selector 'app=central' --field-selector 'status.phase=Running' --output 'jsonpath={.items..metadata.name}' 2>/dev/null)"
+    central_pod="$(${KUBE_COMMAND} get pod -n "$NAMESPACE" --selector 'app=central' --field-selector 'status.phase=Running' --output 'jsonpath={.items..metadata.name}' 2>/dev/null)"
```

Comment:
```suggestion
    central_pod="$(${KUBE_COMMAND} get pod -n "${NAMESPACE}" --selector 'app=central' --field-selector 'status.phase=Running' --output 'jsonpath={.items..metadata.name}' 2>/dev/null)"
```

Let's be consistent about the use of "$VAR" -> "${VAR}". Could you update this in the remainder of the PR?

Reply:
done